editors/sc-ide/CMakeLists.txt

```diff
@@ -350,21 +350,29 @@ if(APPLE)
                 #fixup_qt5_executable(\"${CMAKE_INSTALL_PREFIX}/SuperCollider/SuperCollider.app\")
                 " COMPONENT Runtime)
 
-
+elseif(CMAKE_GENERATOR MATCHES "Visual Studio")
+  set_property(TARGET SuperCollider PROPERTY RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/editors/sc-ide")
+  install(TARGETS SuperCollider
+      CONFIGURATIONS Debug;Release
+      DESTINATION "SuperCollider"
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+  install(FILES ${translations}
+      CONFIGURATIONS Debug;Release
```

Comment:
as for portaudio, there are two options:

link it as dll and don't care about window/static linking issues (matching compiler options, crt and the like)

use an imported library, like:
```
add_library(portaudio STATIC IMPORTED GLOBAL)

set_target_properties(portaudio PROPERTIES
  IMPORTED_LOCATION_DEBUG   /path/to/portaudio_debug.lib
  IMPORTED_LOCATION                  /path/to/portaudio_release.lib
)
```

portaudio-as-dll might be the most robust way. embeding portaudio into the sc build process shouldn't be too difficult, either, someone just needs to do it.